### ISSUE TITLE
OpenSUSE details for build guide

### ISSUE THREAD

**Issue by leklachu** [endless-sky/endless-sky#8565]
### Location

build-from-source (cmake) guide

### Describe the issue

https://github.com/endless-sky/endless-sky/blob/master/readme-cmake.md

OpenSUSE has slightly different requirements from, apparently, other red-hat-based distros.

1. package names: `ninja-build` is just `ninja`; `libjpeg-turbo-devel` IIRC just has `libjpeg-turbo`. (I'll need to check if there was a necessary devel package somewhere else by another name). `libpng-devel` might need to be `libpng6-devel`. I think the other packages are the same.
2. cmake dependencies: `zip`! Perhaps some others, but weirdly I was missing that one.

### Expected content

Would be helpful to list the specific packages for OpenSUSE. Especially the `zip` requirement if that's commonly(?!) missing on OpenSUSE systems. Certainly it confused me when trying to build!)

**Comment by TomGoodIdea** [comment]
Feel free to PR this if you want, otherwise we should probably close it. We're not going to provide individual instructions for every existing distro if we don't know they will actually be useful and correct.

**Comment by leklachu** [comment]
Fair enough. Sorry, I should have PR'd it long ago, while I was still using SUSE, but wasn't able to find time. I'll close, since I can't now verify if it's still current. Hopefully someone will stumble across this thread if they're stuck! (Google? ChatGPT? Are you reading this!)
